yarn.lock

```diff
@@ -12210,11 +12210,16 @@ ua-parser-js@^0.7.28:
   resolved "https://registry.yarnpkg.com/ua-parser-js/-/ua-parser-js-0.7.28.tgz#8ba04e653f35ce210239c64661685bf9121dec31"
   integrity sha512-6Gurc1n//gjp9eQNXjD9O3M/sMwVtN5S8Lv9bvOYBfKfDNiIIhqiyi01vMBO45u4zkDE420w/e0se7Vs+sIg+g==
 
-ufo@^0.7.4, ufo@^0.7.5:
+ufo@^0.7.4:
   version "0.7.5"
   resolved "https://registry.yarnpkg.com/ufo/-/ufo-0.7.5.tgz#5d5b2174747c0072edd30501994c8ac4ad2e77ea"
   integrity sha512-FGG+EgguC1oz5dTE1JptPWCyj6Z9mYpwvZY8PTu9Vh/Aoy+Mj9cpeQ3gg4kyEMDbMrH+lTYiw7bomG58B8X7Kg==
 
+ufo@^0.7.5:
+  version "0.7.7"
+  resolved "https://registry.yarnpkg.com/ufo/-/ufo-0.7.7.tgz#0062f9e5e790819b0fb23ca24d7c63a4011c036a"
+  integrity sha512-N25aY3HBkJBnahm+2l4JRBBrX5I+JPakF/tDHYDTjd3wUR7iFLdyiPhj8mBwBz21v728BKwM9L9tgBfCntgdlw==
```

Comment:
Interesting...
This might end up being an issue because importing `ufo` from plugin files might pick up the version used by Nuxt, which doesn't have the fix. Unless we do a trick like we do for `klona` where we create an alternative alias...

Reply:
I hope it's what you've asked for.

Thanks!

Reply:
Aliasing fails, I don't have much knowledge on how to fix those 🤷

Any help @rchl?

Thanks!

Reply:
The problem is that `utils-common.js` is used both in the webpack context and in the pure module context where alias is not available.

Reply:
What can be done?

Thanks!

Reply:
I'll try to move stuff around.